frontend/src/css/cat-theme.css

```diff
@@ -196,5 +196,5 @@ Progess Steps
   left: 50%;
   transform: translate(-50%, -50%) !important;
   min-width: 500px !important;
-  max-width: 750px !important;
+  max-width: 1000px !important;
```

Comment:
Is this still required?

Reply:
Fixed in commit #https://github.com/code-for-canada/project-thundercat/pull/178/commits/85f076d65301c286f95812f3b434c75b7f6ae2c6